sql-statements/sql-statement-drop-binding.md

```diff
@@ -15,18 +15,26 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-drop-binding/']
 ```ebnf+diagram
 DropBindingStmt ::=
     'DROP' GlobalScope 'BINDING' 'FOR' ( BindableStmt ( 'USING' BindableStmt )?
-|   'SQL' 'DIGEST' SqlDigest)
+|   'SQL' 'DIGEST' StringLiteralOrUserVariableList)
 
 GlobalScope ::=
     ( 'GLOBAL' | 'SESSION' )?
 
 BindableStmt ::=
     ( SelectStmt | UpdateStmt | InsertIntoStmt | ReplaceIntoStmt | DeleteStmt )
+
+StringLiteralOrUserVariableList:
+    (StringLitOrUserVariable | StringLiteralOrUserVariableList ',' StringLitOrUserVariable)
+
+StringLiteralOrUserVariable:
+    stringLiteral | UserVariable
```

Comment:
It seems that ebnf syntax error exists. You can preview it using [https://kennytm.github.io/website-docs/dist/](https://kennytm.github.io/website-docs/dist/).
```suggestion
StringLiteralOrUserVariableList ::=
    ( StringLitOrUserVariable | StringLiteralOrUserVariableList ',' StringLitOrUserVariable )

StringLiteralOrUserVariable ::=
    ( stringLiteral | UserVariable )
```